app/models/zillow.rb

```diff
@@ -0,0 +1,41 @@
+class Zillow
+  attr_accessor :address, :zpid, :city, :state, :zip, :edited_facts, :links, :images, :zestimate
+
+
+  def initialize(attrs)
+    @address = attrs["updatedPropertyDetails"]["response"]["address"]
+    @zpid    = attrs["zpid"]
+    @city    = attrs["city"]
+    @edited_facts = attrs["updatedPropertyDetails"]["response"]["editedFacts"]
+    @links = attrs["updatedPropertyDetails"]["response"]["links"]
+    @images = attrs["updatedPropertyDetails"]["response"]["images"]
+  end
+
+
+
+  def self.find
+    response = HTTParty.get("http://www.zillow.com/webservice/GetUpdatedPropertyDetails.htm?zws-id=X1-ZWz19vnw1f09vv_82rpe&zpid=48749425")
+    z = self.new response.parsed_response
+    z.zestimate = HTTParty.get("http://www.zillow.com/webservice/GetZestimate.htm?zws-id=X1-ZWz19vnw1f09vv_82rpe&zpid=48749425")["zestimate"]["response"]["zestimate"]["amount"]["__content__"]
```

Comment:
When the time comes, you'll want to refactor this a bit, just to break it into different methods for readability.